model/src/test/java/mops/fragen/FrageTest.java

```diff
@@ -0,0 +1,50 @@
+package mops.fragen;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+public class FrageTest {
+
+  /**
+   * All the classes
+   * MultipleChoiceFrage
+   * MultipleResponseFrage
+   * SingleResponseFrage
+   * TextFrage
+   * Should be tested seperately since Frage is an abstract class.
+   */
+
+  @Test
+  public void aendereOeffentlichkeitsStatusMultipleChoiceFrageTest() {
+    Frage multiplechoicefrage = new MultipleChoiceFrage("x");
+    boolean before = multiplechoicefrage.isOeffentlich();
+    multiplechoicefrage.aendereOeffentlichkeitsStatus();
+    Assertions.assertTrue(before != multiplechoicefrage.isOeffentlich());
+  }
+
+  @Test
+  public void aendereOeffentlichkeitsStatusMultipleResponseFrageTest() {
+    Frage multipleresponsefrage = new MultipleResponseFrage(0L, "x");
+    boolean before = multipleresponsefrage.isOeffentlich();
+    multipleresponsefrage.aendereOeffentlichkeitsStatus();
+    Assertions.assertTrue(before != multipleresponsefrage.isOeffentlich());
```

Comment:
NO! You should use assertTrue for asserting the boolean function isOeffentlich() directly:
```suggestion
    multipleresponsefrage.aendereOeffentlichkeitsStatus();
    Assertions.assertTrue(multipleresponsefrage.isOeffentlich());
```

Reply:
Isn't the idea here is the check that the status has changed by saving the previous status, because without it you'd assume that your function flags isOeffentlich as false initially, aren't the tests supposed to be self contained in the sense that they're understandable without taking any look into the code?
I could be wrong on many aspects here so take it with a grain of salt!

Reply:
The field in the question class is initialized with false. If you change it, it is true. So you can use assertTrue. The answers should be be private until the lecturer makes them public.